homeassistant/components/wyoming/assist_satellite.py

```diff
@@ -782,17 +782,22 @@ async def _stream_tts(self, tts_result: tts.ResultStream) -> None:
                 assert sample_width is not None
                 assert sample_channels is not None
 
-                audio_chunk = AudioChunk(
-                    rate=sample_rate,
-                    width=sample_width,
-                    channels=sample_channels,
-                    audio=data_chunk,
-                    timestamp=timestamp,
-                )
+                data_chunk_idx = 0
+                while data_chunk_idx < len(data_chunk):
```

Comment:
You can use [`islice`](https://docs.python.org/3/library/itertools.html#itertools.islice)